front-end/src/components/molecules/side-bar/side-bar.tsx

```diff
@@ -40,7 +60,7 @@ export default function Sidebar() {
         <div className="w-[90%] h-[35px] rounded-sm text-tertiary-color">
           <span>Logout</span>
         </div>
-      </Link>
+      </button>
```

Comment:
This is a very simple button, and nobody needs some many unused nodes (HTML tags) on the DOM tree.

you should literally have just 2 components within this `<button>` tag.

the issue raised here #50  was addressing this problem.
you might as well just resolve it in this PR...